MQTicketService/TicketGenerator/com/ibm/mq/badge/Manager.java

```diff
@@ -75,6 +75,9 @@ public static void main(String[] args) {
                     logger.info("-----");
                     logger.info("Publishing advert...");
 
+                    final String key = "org.glassfish.jaxb.runtime.v2.bytecode.ClassTailor.noOptimize";
```

Comment:
Add in-code documentation as to what 
```java
 final String key = "org.glassfish.jaxb.runtime.v2.bytecode.ClassTailor.noOptimize";
                     System.setProperty(key, "true");
```
does.

Reply:
added comment to explain what the code does.